From b8ef6feb7bf47232bd58b4328097dc9422b5715e Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 22 Sep 2014 10:36:32 +0200
Subject: [PATCH 14/23] Disable DNSv6 probing

Default to using system defaults regarding IPv6 and IPv4.
No new options to switch autodetection on again are added,
and you can use the existing --disable-ipv6 if need be.
---
 chrome/browser/io_thread.cc | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/chrome/browser/io_thread.cc b/chrome/browser/io_thread.cc
index 883e6f0..85cd184 100644
--- a/chrome/browser/io_thread.cc
+++ b/chrome/browser/io_thread.cc
@@ -195,10 +195,9 @@ scoped_ptr<net::HostResolver> CreateGlobalHostResolver(net::NetLog* net_log) {
 #endif
 
   // Determine if we should disable IPv6 support.
+  global_host_resolver->SetDefaultAddressFamily(
+      net::ADDRESS_FAMILY_UNSPECIFIED); /* disable probing: use both v6 and v4 by default */
   if (command_line.HasSwitch(switches::kEnableIPv6)) {
-    // Disable IPv6 probing.
-    global_host_resolver->SetDefaultAddressFamily(
-        net::ADDRESS_FAMILY_UNSPECIFIED);
   } else if (command_line.HasSwitch(switches::kDisableIPv6)) {
     global_host_resolver->SetDefaultAddressFamily(net::ADDRESS_FAMILY_IPV4);
   }
-- 
2.4.0

